src/lib/Components/Bidding/Components/Button.tsx

```diff
@@ -21,7 +27,7 @@ export class BidGhostButton extends React.Component<ButtonProps> {
 
 const styles = StyleSheet.create({
   default: {
-    height: 46,
+    height: 50,
   },
```

Comment:
We can't get rid of this non-styled CSS definition entirely due to the fact that the `<Button>` component is not built based on the styled-system.